tokio/src/runtime/thread_pool/tests/loom_pool.rs

```diff
@@ -235,6 +241,29 @@ fn gated2(thread: bool) -> impl Future<Output = &'static str> {
     })
 }
 
+/// Fake runtime
+struct Runtime {
+    executor: ThreadPool,
+    #[allow(dead_code)]
+    blocking_pool: crate::blocking::BlockingPool,
```

Comment:
Just held to avoid shutdown.

Reply:
could be 
```suggestion
    _blocking_pool: crate::blocking::BlockingPool,
```
rather than using the allow attr?

Reply:
:shrug: switching to `_blocking_pool` prevents being able to use the short hand for initialization. I opted to throw on an allow over reworking the test more.